src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IIfStatement.vb

```diff
@@ -0,0 +1,143 @@
+          Left: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+          Right: ILiteralExpression (Text: 20) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 20)
+      IBlockStatement (1 statements) (OperationKind.BlockStatement)
+        IExpressionStatement (OperationKind.ExpressionStatement)
+          IAssignmentExpression (OperationKind.AssignmentExpression, Type: System.Int32)
+            Left: ILocalReferenceExpression: returnValue (OperationKind.LocalReferenceExpression, Type: System.Int32)
+            Right: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of SingleLineIfStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17601, "https://github.com/dotnet/roslyn/issues/17601")>
+        Public Sub IIfstatementWithOperator()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim m As Integer = 12
+        Dim n As Integer = 18
+        Dim returnValue As Integer = -1
+        If (m > 10 And n > 20) Then returnValue = n'BIND:"If (m > 10 And n > 20) Then returnValue = n"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IIfStatement (OperationKind.IfStatement)
+  Condition: IParenthesizedExpression (OperationKind.ParenthesizedExpression, Type: System.Boolean)
+      IBinaryOperatorExpression (BinaryOperationKind.BooleanAnd) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+        Left: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+            Left: ILocalReferenceExpression: m (OperationKind.LocalReferenceExpression, Type: System.Int32)
+            Right: ILiteralExpression (Text: 10) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 10)
+        Right: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+            Left: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+            Right: ILiteralExpression (Text: 20) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 20)
+  IBlockStatement (1 statements) (OperationKind.BlockStatement)
+    IExpressionStatement (OperationKind.ExpressionStatement)
+      IAssignmentExpression (OperationKind.AssignmentExpression, Type: System.Int32)
+        Left: ILocalReferenceExpression: returnValue (OperationKind.LocalReferenceExpression, Type: System.Int32)
+        Right: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of SingleLineIfStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
```

Comment:
Please port the test for `ElseIf` from C# to VB.

As a general guideline, it is recommended that you try to port all the C# tests to VB or vice versa for equivalent features.